### PR TITLE
Jwilaby/#696 blank menu item mac

### PR DESCRIPTION
**Pull request by justinwilaby**

### CODE REVIEW

File: packages/app/main/src/appMenuBuilder.ts [hunk 1]
```diff
@@ -254,9 +254,13 @@ export const AppMenuBuilder = new class AppMenuBuilderImpl implements AppMenuBui
...
         { role: 'paste' },
+        process.platform === 'darwin' ? {
+          role: 'pasteAndMatchStyle'
+        } : null,
         { role: 'delete' },
-        { role: 'selectall' },
-      ]
+        process.platform === 'win32' ? { type: 'separator' } : null,
+        { role: 'selectall' }
+      ].filter(item => item) as any[]
```
Comment (tonyanziano) [inline packages/app/main/src/appMenuBuilder.ts]: So where was the blank item coming from? Was it just an unfiltered item? Or was it the selectall?

Comment (justinwilaby) [reply packages/app/main/src/appMenuBuilder.ts]: I think it was the select all `selectAll` does not work while `selectall` does. also, it looks like separators are conditional based on the platform and where they fall in the menu item ordering.

Comment (tonyanziano) [reply packages/app/main/src/appMenuBuilder.ts]: Ah I see. 👍

Comment (justinwilaby) [reply packages/app/main/src/appMenuBuilder.ts]: Full disclosure, this hasn't been tested on a Mac per se, but the interwebs say this is on fleek!

Comment (justinwilaby) [reply packages/app/main/src/appMenuBuilder.ts]: image

**Pull request merged**